gradle/dependencies.gradle

```diff
@@ -45,7 +45,8 @@ def kotlin = [
 ]
 
 def misc = [
-    errorProneAnnotations: "com.google.errorprone:error_prone_annotations:${versions.errorProne}"
+    errorProneAnnotations: "com.google.errorprone:error_prone_annotations:${versions.errorProne}",
+    jsr305: 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
Add spaces after the jsr305 to match the width till the `:` on the previous line